e2e/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.19
```

Comment:
What do you think about running the non-private tests containerized like this as well? Even though it might be a bit more brittle in general, seems like standardizing on one approach for private/non-private is a good enough justification.

Reply:
I think this is the right way as well. It makes it so the point of these e2e tests are solely to test the WAR code while rp tests tests ensure the code there.

Reply:
I don't see why not! Can definitely look into doing this